src/data/quickstart-metadata.json

```
@@ -0,0 +1,7 @@
+[
+  {
+    "slug": "example",
+    "title": "meta title",
+    "description": "meta description"
+  }
+]
```

To make it easier to look up a quickstart, we could use the slug as the key.
```suggestion
{
  "example": {
    "title": "meta title",
    "description": "meta description"
  }
}
```